README

```diff
@@ -679,31 +679,64 @@ Network Support
       or
       shell$ mpirun --mca pml cm ...
 
+  - UCX is the Unified Communication X (UCX) communication library
+    (http://www.openucx.org/).  This is an open-source project
+    developed in collaboration between industry, laboratories, and
+    academia to create an open-source production grade communication
+    framework for data centric and high-performance applications.  The
+    UCX library can be downloaded from repositories (e.g.,
+    Fedora/RedHat yum repositories).  The UCX library is also part of
+    Mellanox OFED and Mellanox HPC-X binary distributions.
+
+    UCX currently supports:
+
+    - OpenFabrics Verbs (including InfiniBand and RoCE)
+    - Cray's uGNI
+    - TCP
+    - Shared memory
+    - NVIDIA CUDA drivers
+
+  - "yalla" uses the Mellanox MXM transport library.  MXM is the
+     Mellanox Messaging Accelerator library, utilizing a full range of
+     IB transports to provide the following messaging services to the
+     upper level MPI/OpenSHMEM libraries:
+
+     - Usage of all available IB transports
+     - Native RDMA support
+     - Progress thread
+     - Shared memory communication
+     - Hardware-assisted reliability
+
 - Similarly, there are two OpenSHMEM network models available: "ucx",
   and "ikrit":
   - "ucx" interfaces directly with UCX;
   - "ikrit" interfaces directly with Mellanox MXM.
 
-- UCX is the Unified Communication X (UCX) communication library
-  (http://www.openucx.org/).
-  This is an open-source project developed in collaboration between
-  industry, laboratories, and academia to create an open-source
-  production grade communication framework for data centric and
-  high-performance applications.
-  UCX currently supports:
-  - OFA Verbs;
-  - Cray's uGNI;
-  - NVIDIA CUDA drivers.
-
-- MXM is the Mellanox Messaging Accelerator library utilizing a full
-  range of IB transports to provide the following messaging services
-  to the upper level MPI/OpenSHMEM libraries:
-
-  - Usage of all available IB transports
-  - Native RDMA support
-  - Progress thread
-  - Shared memory communication
-  - Hardware-assisted reliability
+- In prior versions of Open MPI, InfiniBand and RoCE support was
+  provided through the openib BTL and ob1 PML plugins.  Starting with
+  Open MPI 4.0.0, InfiniBand support through the openib+ob1 plugins is
+  both deprecated and superseded by the ucx PML component.
+
+  While the openib BTL depended on libibverbs, the UCX PML depends on
+  the UCX library.
+
+  Once installed, Open MPI can be built with UCX support by adding
+  --with-ucx to the Open MPI configure command. Once Open MPI is
+  configured to use UCX, the runtime will automatically select the UCX
+  PML if one of the supported networks is detected (e.g., InfiniBand).
+  It's possible to force using UCX in the mpirun or oshrun command
+  lines by specifying any or all of the following mca parameters:
+  "-mca pml ucx" for MPI point-to-point operations, "-mca spml ucx"
```

Comment:
guess need to use "--mca" here as well (instead of -mca)?

Reply:
Good catch; will fix.